### ISSUE TITLE
Powietrze 1.4.2 doesn't work event don't want to start

### ISSUE THREAD

**Issue by atlochowski** [Kormil/harbour-powietrze#14]
```
Apr 07 08:17:36 Sailfish lipstick[1631]: GPS created
Apr 07 08:17:36 Sailfish lipstick[1631]: Models created
Apr 07 08:17:37 Sailfish lipstick[1631]: Providers connection created
Apr 07 08:17:37 Sailfish lipstick[1631]: Providers created
Apr 07 08:17:37 Sailfish lipstick[1631]: Models settings loaded
Apr 07 08:17:37 Sailfish lipstick[1631]: GPS REQUEST
Apr 07 08:17:37 Sailfish booster-silica-qt5[13657]: [D] unknown:0 - Using Wayland-EGL
Apr 07 08:17:37 Sailfish [13581]: [D] unknown:0 - Geoclue client path: "/org/freedesktop/Geoclue/Master/client24"
Apr 07 08:17:37 Sailfish dbus-daemon[1524]: dbus-daemon[1524]: [session uid=100000 pid=1524] Activating via systemd: service name='org.freedesktop.Geoclue.Providers.Hybris' unit='dbus-org.freedesktop.Geoclue.Providers.Hybris.service' requested by ':1.109' (uid=100000 pid=5656 comm="/usr/libexec/geoclue-master")
Apr 07 08:17:37 Sailfish systemd[1459]: Starting Geoinformation Service Hybris Provider...
Apr 07 08:17:37 Sailfish lipstick[1631]: GPS position: 52.1561 : 21.05
Apr 07 08:17:37 Sailfish kernel: harbour-powietr: unhandled page fault (11) at 0x0000000c, code 0x005
Apr 07 08:17:37 Sailfish kernel: pgd = d0dd4000
Apr 07 08:17:37 Sailfish kernel: [0000000c] *pgd=00000000
Apr 07 08:17:37 Sailfish kernel: 
Apr 07 08:17:37 Sailfish kernel: Pid: 13581, comm:      harbour-powietr
Apr 07 08:17:37 Sailfish kernel: CPU: 0    Tainted: P        W  O  (3.4.108.20191219.1 #1)
Apr 07 08:17:37 Sailfish kernel: PC is at 0x4295fa44
Apr 07 08:17:37 Sailfish kernel: LR is at 0x42970915
Apr 07 08:17:37 Sailfish kernel: pc : [<4295fa44>]    lr : [<42970915>]    psr: 600e0030
                                 sp : bee1ee88  ip : 00000000  fp : 429e2000
Apr 07 08:17:37 Sailfish kernel: r10: 429708f1  r9 : bee1eed4  r8 : 401098c4
Apr 07 08:17:37 Sailfish kernel: r7 : 401098c4  r6 : 45f5bd60  r5 : 429e2000  r4 : bee1ee88
Apr 07 08:17:37 Sailfish kernel: r3 : 00000000  r2 : 00000000  r1 : 00000000  r0 : 00000000
Apr 07 08:17:37 Sailfish kernel: Flags: nZCv  IRQs on  FIQs on  Mode USER_32  ISA Thumb  Segment user
Apr 07 08:17:37 Sailfish kernel: Control: 10c5787d  Table: 90fd406a  DAC: 00000015
Apr 07 08:17:37 Sailfish kernel: [<c010b258>] (unwind_backtrace+0x0/0x120) from [<c010ef6c>] (__do_user_fault+0x74/0xbc)
Apr 07 08:17:37 Sailfish kernel: [<c010ef6c>] (__do_user_fault+0x74/0xbc) from [<c0887f80>] (do_page_fault+0x374/0x3b4)
Apr 07 08:17:37 Sailfish kernel: [<c0887f80>] (do_page_fault+0x374/0x3b4) from [<c01002d4>] (do_DataAbort+0x138/0x1a8)
Apr 07 08:17:37 Sailfish kernel: [<c01002d4>] (do_DataAbort+0x138/0x1a8) from [<c0886814>] (__dabt_usr+0x34/0x40)
Apr 07 08:17:37 Sailfish kernel: Exception stack(0xd37e5fb0 to 0xd37e5ff8)
Apr 07 08:17:37 Sailfish kernel: 5fa0:                                     00000000 00000000 00000000 00000000
Apr 07 08:17:37 Sailfish kernel: 5fc0: bee1ee88 429e2000 45f5bd60 401098c4 401098c4 bee1eed4 429708f1 429e2000
Apr 07 08:17:37 Sailfish kernel: 5fe0: 00000000 bee1ee88 42970915 4295fa44 600e0030 ffffffff
Apr 07 08:17:37 Sailfish kernel: harbour-powietr(13581) send signal 11 to harbour-powietr(13581)
Apr 07 08:17:38 Sailfish mapplauncherd[1718]: Boosted process (pid=13581) was terminated due to signal 11
Apr 07 08:17:38 Sailfish invoker[13649]: error: Can't send signal 11 to application [13581]: No such process
Apr 07 08:17:38 Sailfish kernel: booster-silica-(1718) send signal 11 to invoker(13649)
Apr 07 08:17:38 Sailfish kernel: invoker(13649) send signal 11 to invoker(13649)
Apr 07 08:17:38 Sailfish mapplauncherd[13657]: SilicaBooster: Initiate asynchronous preload.
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: [W] unknown:0 - Android GPS hardware module "gps" "loc_api GPS Module" 1.0
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: [W] unknown:0 - Initialising GPS interface
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: xport_lookup(383) QMUXD: Service_id=16 not found over conn_id=9
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: xport_lookup(383) QMUXD: Service_id=16 not found over conn_id=9
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: xport_lookup(383) QMUXD: Service_id=16 not found over conn_id=9
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: xport_lookup(383) QMUXD: Service_id=16 not found over conn_id=9
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: xport_lookup(383) QMUXD: Service_id=16 not found over conn_id=9
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: [W] unknown:0 - Initialising AGPS Interface
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: [W] unknown:0 - Initialising GPS NI Interface
Apr 07 08:17:39 Sailfish geoclue-hybris[13659]: [W] unknown:0 - Initialising GPS Xtra Interface
Apr 07 08:17:39 Sailfish systemd[1459]: Started Geoinformation Service Hybris Provider.
Apr 07 08:17:39 Sailfish dbus-daemon[1524]: dbus-daemon[1524]: [session uid=100000 pid=1524] Successfully activated service 'org.freedesktop.Geoclue.Providers.Hybris'
Apr 07 08:17:39 Sailfish dbus-daemon[1524]: dbus-daemon[1524]: [session uid=100000 pid=1524] Activating service name='org.freedesktop.Geoclue.Providers.Here' requested by ':1.109' (uid=100000 pid=5656 comm="/usr/libexec/geoclue-master")
Apr 07 08:17:39 Sailfish dbus-daemon[1524]: dbus-daemon[1524]: [session uid=100000 pid=1524] Successfully activated service 'org.freedesktop.Geoclue.Providers.Here'
Apr 07 08:17:39 Sailfish kernel: dbus-daemon(1524) send signal 9 to dbus-daemon(13675)
Apr 07 08:17:56 Sailfish dbus-daemon[747]: [system] Rejected send message, 3 matched rules; type="method_return", sender=":1.5" (uid=0 pid=1043 comm="/usr/sbin/ofonod -n --nobacktrace --noplugin=he910") interface="(unset)" member="(unset)" error name="(unset)" requested_reply="0" destination=":1.13" (uid=0 pid=1127 comm="/usr/bin/fastdormancyd")
Apr 07 08:17:57 Sailfish dbus-daemon[747]: dbus-daemon[747]: [system] Rejected send message, 3 matched rules; type="method_return", sender=":1.5" (uid=0 pid=1043 comm="/usr/sbin/ofonod -n --nobacktrace --noplugin=he910") interface="(unset)" member="(unset)" error name="(unset)" requested_reply="0" destination=":1.13" (uid=0 pid=1127 comm="/usr/bin/fastdormancyd")

```

**Comment by atlochowski** [comment]
on Jolla 1 with 3.3.0.14. With previous version of Powietrze everything was ok.

**Comment by Kormil** [comment]
Thank you for report!

**Comment by Kormil** [comment]
Fixed in 1.4.3
